M2/configure.ac

```diff
@@ -741,6 +742,18 @@ AC_COMPILE_IFELSE([AC_LANG_SOURCE([
 	# 1 "foo" 1 3 4
 	template <class T> struct A { };])],AC_MSG_RESULT(no);ISYSTEM=yes,AC_MSG_RESULT(yes))
 
+dnl Check for openmp and fail if it cannot be found.
+dnl This is required for building csdp and good for normaliz
+AC_MSG_CHECKING(whether OpenMP is supported by the compiler)
+AC_LANG(C++)
+AC_PROG_CXX
+AC_OPENMP
+if test "$ac_cv_prog_cxx_openmp" = unsupported
+   then AC_MSG_ERROR([OpenMP does not work, use a compiler that supports OpenMP])
+fi
+AC_SUBST(OPENMP_CXXFLAGS,"$OPENMP_CXXFLAGS")
+AC_SUBST(OPENMP_LDFLAGS,"")
```

Comment:
What's the purpose of this?

Reply:
What is the purpose of what?  I don't understand your question.  Do you mean just the AC_SUBST(OPENMP_LDFLAGS, "") line, or the whole thing (which we have benn discussing the need for in the last week).

Reply:
Yes, what is the purpose of adding OPENMP_LDFLAGS if its value is always empty?